### ISSUE TITLE
extend functionality of crop_edges

### ISSUE THREAD

**Issue by jlaehne** [LumiSpy/lumispy#129]
#### Describe the context
`crop_edges` is a wrapper function for `inav` that takes the parameter `crop_px` and removes as many pixels from all sides of a map.

#### Describe the functionality you would like to see.
To make this function more versatile, it would make sense to alternatively allow the following sets of parameters:
- `crop_x_px` and `crop_y_px` to crop different amounts of pixels from the x and y directions
- `crop_left_px`, `crop_right_px`, `crop_top_px`, `crop_bottom_px` to crop different amounts of pixels from all sides (check in `matplotlib` what order would be typically used for such a set of parameters corresponding to the edges).

#### Additional information
@jordiferrero , you originally implemented this function. Would this extension make sense to you? We have a student that could implement it.

**Comment by jordiferrero** [comment]
This makes sense.
I initially implemented this function because I always run this "simple" cropping during pre-processing.
It would be great for a student to do it. Just make sure the original functionality is still working (so when you only pass 1 parameter, it crops all sides equally).

**Comment by jlaehne** [comment]
Of course, one should be able to pass either of the three sets of parameters mentioned above.

**Comment by jordiferrero** [comment]
Glad to hear a student is coming to help you with coding :-)
